app/views/users/_profile.html.erb

```diff
@@ -9,12 +9,24 @@
 
     <div class="crayons-field">
       <%= f.label :name, for: "user[name]", class: "crayons-field__label" %>
-      <%= f.text_field "user[name]", maxlength: 30, class: "crayons-textfield", placeholder: "John Doe", value: @user.name %>
+      <%= f.text_field "user[name]",
+                       maxlength: 30,
+                       class: "crayons-textfield",
+                       placeholder: "John Doe",
+                       value: @user.name,
+                       data: { character_span_id: "name-characters" } %>
+      <div class="align-self-flex-end fs-s"><span id="name-characters"></span>/30</div>
```

Comment:
```suggestion
      <div class="self-end fs-s"><span id="name-characters"></span>/30</div>
```

I think you can use this helper class instead of having to create a new one 🙂

Reply:
I've been there a million times @msarit 😅